website/src/components/Header/NavLinks.tsx

```diff
@@ -14,8 +15,8 @@ export function NavLinks(): JSX.Element {
   return (
     <>
       {[
-        ["Join Us", "/#join-us"],
         ["FAQ", "/#faq"],
+        ["Join Us", "/#join-us"],
```

Comment:
Right now these links don't reliably go anywhere.  For now, can you drop usage of this `NavLinks` component in `Header`?